challenges/tests.py

```diff
@@ -1,3 +1 @@
-from django.test import TestCase
```

Comment:
Jak plik jest pusty to możesz go wywalić ;)

Reply:
👍🏻